test/cluster-kind.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+name: wpa-e2e-test
+networking:
+  apiServerPort: 32001
```

Comment:
is it necessary?

Reply:
I think this is leftover from the previous e2e, not necessary I think.